terraform/aws/output.tf

```diff
@@ -3,12 +3,12 @@ output "public_ip" {
   description = "The public IPs address of the provisioned node(s)"
 }
 
-output "private_dns" {
+output "instance_private_dns" {
```

Comment:
why do we prepend this output with `instance_` and don't do the same for `private_ips`?